### ISSUE TITLE
Gem 'cancancan' is not available in SDK 'ruby-2.0.0-p596'

### ISSUE THREAD

**Issue by AfolabiOlaoluwa** [CanCanCommunity/cancancan#343]
Anytime I add gem 'cancancan' to my application and bundle install, it doesn't work and I get the error Gem 'cancancan' is not available in SDK 'ruby-2.0.0-p596'. In other words, it doesn't want to install or not just available for Ruby 2.0.0.

How do I fix this problem without having to install another version of Ruby?
1. Am using Ruby 2.0.0
2. Am using Rails 4.2.1
3. Am using IDE RubyMine.

Note: I have done the RubyMine fix to the problem/error which mean interpreter not available, but all to no avail. Other gems are installing well and working well.

Is it that Cancancan requires a minimum version of Ruby to use with it?

**Comment by alexsanford** [comment]
I installed `ruby-2.0.0-p648` through RVM, added `gem 'cancancan'` to the Gemfile, and ran `bundle install` and it worked fine for me. I couldn't specifically install `p596` but I don't think it should make a difference.

What version of bundler are you using? `bundler --version`

Are you using standard RubyGems? `source 'https://rubygems.org` in your Gemfile?

**Comment by AfolabiOlaoluwa** [comment]
I can tell am using Bundler version 1.10. I even upgraded to the newest version 1.13, yet I still have the same error. 
I can also tell that am using the standard RubyGem as typed. 
I think the problem probably is from RubyMine or Cancancan Gem. Though am not sure. I tried all the a RubyMine suggested fixes since they understand that error to mean I didn't select my Ruby interpreter to man my application, which is not true. As I stated in the lodged error, some gems install well and others don't. 

Note am on Windows environment.
